### PR TITLE
Preserve user profile privacy

### PR DESCRIPTION
**Pull request by Tenrys**
I've been added by random people many times before, who just saw me show up on the website thanks to being in the PreMiD Discord, which I find weird and irritating, so at least randomize the user discriminator so you can't be added that easily, while still looking like a real user.

### CODE REVIEW

**Comment by EncryptedDev** [comment]
I don’t see any point in this. If you’re in the PreMiD guild, this is accessible data by viewing the member list - you could be “lucky” to be displayed on the website, but there is no reason to hide your identity for this…

**Comment by Tenrys** [comment]
I've been "lucky" enough apparently then, people are weird like that and add others they don't know which are brought forward on a product's landing page, instead of picking out from a member list.

File: src/pages/index.vue [hunk 1]
```diff
@@ -440,7 +440,7 @@
 
 				presence.profile = {
 					name: this.users[index].user.name,
-					discriminator: this.users[index].user.tag,
+					discriminator: [,,,,].fill().map(() => Math.floor(Math.random() * 10)).join(""),
```
Comment (ImRodry) [inline src/pages/index.vue:443]:
```suggestion
					discriminator: Array(4).fill().map(() => Math.floor(Math.random() * 10)).join(""),
```

Comment (Tenrys) [reply src/pages/index.vue:443]: It's the same thing, just longer? Unless you think this looks better style wise?

Comment (EncryptedDev) [reply src/pages/index.vue:443]: It looks way better

**Comment by Tenrys** [comment]
This happened to me again today, someone just adding me to see if the displayed nick was real. I guess I'll leave the premid Discord if this doesn't get merged.

**Comment by ImRodry** [comment]
And why is that an issue? I’ve never had that happen to me but if I got a friend request from an unknown user I’d just ignore it. Not only that but users love to see themselves on the website and that makes them feel special so why would we fake the tag?

**Comment by callumok2004** [comment]
I have shown up on that section at least 100 times in the past 2-3 years, and not once has someone asked me to check if it is real.
I see no need for this commit to be merged.

**Pull request closed without merging**